core/src/test/java/NoOpLogger.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2018 JUUL Labs, Inc.
+ */
+
+package com.juul.able.experimental
+
+class NoOpLogger : Logger {
+    override fun isLoggable(priority: Int): Boolean = false
+    override fun log(priority: Int, throwable: Throwable?, message: String) {
+        error("No-op logger is not loggable.")
```

Comment:
Is this really no-op? Would it be nicer to just do nothing?

Reply:
IMHO it is better to fail loudly than silently?

_(I'd prefer to know something is implemented incorrectly by crashing, than spend a lot of time wondering what's going on w/o any logs)._

Reply:
Yeah, fair.